content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -308,21 +317,10 @@ data/images/
 $ dvc add data/images
 ```
 
-When running `dvc add` on this directory of images, a `data/images.dvc`
```

Comment:
why this change? it looks more or less good for me

Reply:
I removed this chunk because it was explaining `dvc add`/`.dvc` files, not the cache structure.

I can put it back if you prefer.